tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -757,23 +757,65 @@ def get_network_batch_size(inputs_info):
 def show_available_devices():
     print("\nAvailable target devices:  ", ("  ".join(Core().available_devices)))
 
+def dict2string(config):
```

Comment:
Why do we need one more serializer inside tool?

Reply:
From my understanding, we need to keep the same conversation rules for `ov::device::properties` between string and map as those implemented in the C++ version, so I implemented this serializer here.
https://github.com/openvinotoolkit/openvino/blob/53d9b26e1fd438ed03e1333e1726ea3ecafe582e/src/core/include/openvino/core/any.hpp#L351

Reply:
@yangwang201911 These functions are not needed. You can use somerhing like:
```
            if isinstance(value, properties.hint.PerformanceMode):
                value_string = value.name
            elif isinstance(value, OVAny):
                value_string = str(value.value)
            else:
                value_string = str(value)
```

Reply:
BTW performance mode is broken after your changes
```
TypeError: Object of type PerformanceMode is not JSON serializable

```

Reply:
Thank you for your kind reminder.  I suppose the dumped config for `ov::device::properties` should be like as `"DEVICE_PROPERTIES": "{CPU:{NUM_STREAMS:3},GPU:{NUM_STREAMS:5}}"`, which has same format as implemented in[ `any.hpp`](https://github.com/openvinotoolkit/openvino/blob/53d9b26e1fd438ed03e1333e1726ea3ecafe582e/src/core/include/openvino/core/any.hpp#L351), right? while I got this after apply your suggestion `"DEVICE_PROPERTIES": "{'CPU': {'NUM_STREAMS': '5'}, 'GPU': {'NUM_STREAMS': '9'}}"`. Is this our expected format when dumping/loading properties setting from JSON file?

> @yangwang201911 These functions are not needed. You can use somerhing like:
> 
> ```
>             if isinstance(value, properties.hint.PerformanceMode):
>                 value_string = value.name
>             elif isinstance(value, OVAny):
>                 value_string = str(value.value)
>             else:
>                 value_string = str(value)
> ```

Reply:
@yangwang201911 If the question about quotes and spaces, I think the following line will align format:
```
            if isinstance(value, properties.hint.PerformanceMode):
                value_string = value.name
            elif isinstance(value, OVAny) and (isinstance(value.value, dict)):
                value_string = str(value.value).replace('\'','').replace(' ' , '')
            else:
                value_string = str(value)
```
Anyway additional functions for serializations look excessive

Reply:
updated.